### DISCUSSION TITLE
URL Monitoring

### DISCUSSION THREAD

**Discussion by kisanity** [signalfx/splunk-otel-collector#2196] [Q&A] [answered]
Hello,

Is this the only way to monitor URLs: https://docs.splunk.com/Observability/gdi/http/http.html?

What if we have 100s of URLs that we need to monitor, can we point the config in the Helm values.yaml to a specific file? If so, how? And how would we manage it?

Is there another way to monitor URLs?

We're using the Helm chart on OpenShift.

Thanks

**Comment by atoulme** [comment]
You can use this monitor indeed. You can use helm to help you manage and generate the yaml of the configuration as needed. Feel free to get in touch with your account team to propose ideas to make this easier.

**Comment by atoulme** [comment, marked as answer]
Cross-posting to https://github.com/signalfx/splunk-otel-collector-chart/discussions/569
